ui/components/input_data.py

```diff
@@ -0,0 +1,32 @@
+import streamlit as st
+from rip.utils import validate_url
+
+def input_initial_data():
+    st.write("Which impact are you curious about?")
```

Comment:
#* Finding resources we can use as an input for our solution. (GPT3 Compare for semantic search and recommendations)
    st.subheader("What solution you are thinking about against lowering CO2 emission?")
    title = st.text_input('Your favorite solution:', "I'm planning to purchase an electric car")
    
    #* Grouping, filtering and transforming the inputs which migh be useful for our search on second layer affects of policies and applications. (GPT3 Edit)
    st.subheader("We found those side affects regarding your choice")
    st.write("batteries, aluminium usage, electricity demand, ...")
    
    #* Generating quotes and new short definitions from what we found, as a list of takeaways. (GPT3 Explain and Write)
    st.subheader("List of takeaways which might be considered, according to current policies")
    st.write("* you can look for the production processes which are using less ... for batteries")
    st.write("* those materials could be picked insted of aluminium usage: ...")
    st.write("* to decrease electricity demand, ...")
    
    #* Supporting our findings with auto generated images from the copies we generated. (DALL-E)
    st.subheader("Your DALL-E representation of the takeaways")
    st.image("figs/default_img.png")
    
    
    st.subheader("Which impact are you more curious about?")

Reply:
I was thinking replacing this line
    st.write("Which impact are you curious about?")

with the block I pasted :)